### PR TITLE
Update user_import.md

### PR DESCRIPTION
**Pull request by gsinghAppboy**
# Pull Request/Issue Resolution

**Description of Change:**
> Adding a note that Braze does not sanitize data during the import process.  This is based on a thread shared by Jon@braze.com.  We will also look into adding the 'escape' liquid filter.  Once that filter is added, we can update this blurb to also provide a path forward using liquid.

Closes #**ISSUE_NUMBER_HERE**

### Is this change associated with a Braze feature/product release?
- [ ] Yes (__Insert Feature Release Date Here__)
- [ ] No

> If yes, please note the date of the feature release.

---
---

<details>
<summary>PR Checklist</summary>

- [ ] Check that all links work!
- [ ] Ensure you have completed [our CLA](https://www.braze.com/docs/cla/).
- [ ] Tag @Timothy-Kim and @KellieHawks as a reviewer when your work is _done and ready to be reviewed for merge_.
- [ ] Tag others as Reviewers as necessary.
- [ ] If you have modified any links, be sure to add redirects to `config/nginx.conf.erb`.

</details>

<details>
<summary>Writing & Style Quick Tips</summary>

- Use the formatting shown on the [Styling Test Page](https://github.com/Appboy/braze-docs/blob/develop/_docs/_home/styling_test_page.md) to create unique templates and vehicles for your information.
- Use Action verbs!
- Capitalize proper nouns, unique Braze tools and features, custom Braze channels, and all Header titles.
- Redirect when necessary!
- Always speak to the Braze customer - refer to them as “you”, whether they are an Engineer/Developer, a Marketer, or an Admin.
- On Voice: You should aim to be a Friendly Professor. Imagine, if you will, the smartest, but also kindest professor you had. You know, the one who showed you your potential and tested you, but also went slowly in class so you could take great notes and learn some things? Don't be too peppy, but don't be formal. You’re not selling anything - you’re instructing and informing and enabling.

_Consult the [Docs Text Formatting Guide](https://github.com/Appboy/braze-docs/wiki/Special-Formatting) and the [Docs Writing Style Guide & Best Practices](https://github.com/Appboy/braze-docs/wiki/Writing-Style-Guide-&-Best-Practices) for more details._

</details>

<details>
<summary>ATTENTION: For PR Reviewers</summary>

- [ ] Read our [Reviewing a PR page](https://github.com/Appboy/braze-docs/wiki/Reviewing-a-PR) for more on our reviewing suggestions.
- [ ] Preview all changes in the linked Heroku environment (click `View deployment` button below, then "Docs". A `502` error just means you should refresh until you see the Docs Home page.
</details>

---
---

### CODE REVIEW

**Comment by gsinghAppboy** [comment]
Escape filter story: https://jira.braze.com/browse/EMAIL-1813
Will see if someone on the team can pick it up in the coming sprints.

File: _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md [hunk 1]
```diff
@@ -10,6 +10,10 @@ tool: Dashboard
 
 There are two approaches for importing customer data into your Braze dashboard: REST API and CSV.
 
+{% alert important %}
+Note: Braze does not sanitize data during ingestion time.  While importing data into Braze, specifically meant for personalization usage, ensure that it is stripped of HTML, javascript, or any other script tag that potentially could be leveraged maliciously.
```
Comment (jonhyman) [inline _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md:14]: Thoughts on:

> Note: Braze does not HTML sanitize data during ingestion time.  When importing data into Braze, specifically meant for personalization usage in a web browser, ensure that it is stripped of HTML, JavaScript, or any other script tag that potentially could be leveraged maliciously when rendered in a web browser. Alternatively, you can use Braze's Liquid filters to HTML escape rendered text.

Comment (gsinghAppboy) [reply _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md]: updated.  Also added an additional line that was suggested before "Alternatively, for HTML, you can use Braze's Liquid filters (strip_html) to HTML escape rendered text. "

**Review by jonhyman — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md [hunk 2]
```diff
@@ -10,6 +10,10 @@ tool: Dashboard
 
 There are two approaches for importing customer data into your Braze dashboard: REST API and CSV.
 
+{% alert important %}
+Braze does not HTML sanitize data during ingestion time. When importing data into Braze, specifically meant for personalization usage in a web browser, ensure that it is stripped of HTML, JavaScript, or any other script tag that potentially could be leveraged maliciously when rendered in a web browser.  Alternatively, for HTML, you can use Braze's Liquid filters (strip_html) to HTML escape rendered text.
```
Comment (paxchow) [inline _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md:14]:
```suggestion
Braze does not sanitize HTML data during ingestion time. When importing data into Braze, specifically meant for personalization usage in a web browser, ensure that it is stripped of HTML, JavaScript, or any other script tag that potentially could be leveraged maliciously when rendered in a web browser.  Alternatively, for HTML, you can use Braze's Liquid filters (strip_html) to HTML escape rendered text. 
```

File: _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md [hunk 3]
```diff
@@ -10,6 +10,10 @@ tool: Dashboard
 
 There are two approaches for importing customer data into your Braze dashboard: REST API and CSV.
 
+{% alert important %}
+Braze does not HTML sanitize data during ingestion time. When importing data into Braze, specifically meant for personalization usage in a web browser, ensure that it is stripped of HTML, JavaScript, or any other script tag that potentially could be leveraged maliciously when rendered in a web browser.  Alternatively, for HTML, you can use Braze's Liquid filters (strip_html) to HTML escape rendered text. 
+{% endalert %}
+
```
Comment (paxchow) [inline _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md:16]: Should we add an example?

**Review by paxchow — APPROVED** [review]

File: _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md (hunk 2 above)
Comment (gsinghAppboy) [reply _docs/_user_guide/data_and_analytics/user_data_collection/user_import.md]: added an example and took your verbiage

**Review by paxchow — DISMISSED** [review]

**Review by KellieHawks — APPROVED** [review]

**Pull request merged**